doc/rados/troubleshooting/log-and-debug.rst

```diff
@@ -2,177 +2,204 @@
  Logging and Debugging
 =======================
 
-Typically, when you add debugging to your Ceph configuration, you do so at
-runtime. You can also add Ceph debug logging to your Ceph configuration file if
-you are encountering issues when starting your cluster. You may view Ceph log
-files under ``/var/log/ceph`` (the default location).
-
-.. tip:: When debug output slows down your system, the latency can hide 
-   race conditions.
-
-Logging is resource intensive. If you are encountering a problem in a specific
-area of your cluster, enable logging for that area of the cluster. For example,
-if your OSDs are running fine, but your metadata servers are not, you should
-start by enabling debug logging for the specific metadata server instance(s)
-giving you trouble. Enable logging for each subsystem as needed.
-
-.. important:: Verbose logging can generate over 1GB of data per hour. If your 
-   OS disk reaches its capacity, the node will stop working.
-   
-If you enable or increase the rate of Ceph logging, ensure that you have
-sufficient disk space on your OS disk.  See `Accelerating Log Rotation`_ for
-details on rotating log files. When your system is running well, remove
-unnecessary debugging settings to ensure your cluster runs optimally. Logging
-debug output messages is relatively slow, and a waste of resources when
-operating your cluster.
-
-See `Subsystem, Log and Debug Settings`_ for details on available settings.
+You might want to add debugging to your Ceph configuration. The typical time to
+add debugging is at runtime. You might also want to add Ceph debug logging to
+your Ceph configuration file. Adding debug logging can be useful if you are
+encountering issues when starting your cluster. By default, Ceph log files are
+in ``/var/log/ceph``.
+
+.. tip:: Remember that debug output can slow down your system, and that this
+   latency sometimes hides race conditions.
+
+Debug logging is resource intensive. If you encounter a problem in a specific
+area of your cluster, begin troubleshooting by enabling logging for only that
+area of the cluster. For example, if your OSDs are running without errors, but
+your metadata servers are not, enable logging for any specific metadata server
+instances that are having problems. Continue by enabling logging for each
+subsystem only as needed.
+
+.. important:: Verbose logging sometimes generates over 1 GB of data per hour.
+   If the disk that your operating system runs on (your "OS disk") reaches its
+   capacity, the node associated with that disk will stop working.
+
+Whenever you enable or increase the rate of debug logging, make sure that you
+have sufficient disk space on your OS disk. For details on rotating log files,
```

Comment:
have ample capacity for log files, as this may dramatically increase their size.

With containers, centralized logging, syslog, etc. this wording is more general.

Reply:
Accepted.